### DISCUSSION TITLE
MOTD Customization Support

### DISCUSSION THREAD

**Discussion by GangPop** [Zrips/CMI#6143] [Feature Requests]
Simple. The ability to configure the MOTD with CMI (in-game commands &/or config option). Support for hex codes.

**Comment by mrfloris** [comment]
CMI can do two types of MOTD, one being the welcome message when they enter the game and get a welcomeMessage through CMI's ctext feature (and first time joiners can get a Newbie kit) You can optionally make an essentialsx like command such as /motd so they can also query the welcome msg at a later time in-game. Anyway, the other is the Server List in the client, before you connect. There's a CMI Command for this. And you can use hex colors.

I've got info about both of these motd things in a personal faq i wrote, might be helpful;
<https://github.com/mrfdev/CMI/blob/master/Resources/FAQ/cmi-motd.md>

That said:

```
/cmi setmotd <MSG> -s
```
You can use this, use hex colors, and multiple lines using `\n`

Examples:
```
# hex color
cmi setmotd {#orange}Join us for awesome survival

# multiple colors
cmi setmotd {#67a712}Hello {#red} how are you

# gradient from red to yellow
cmi setmotd {#red>} Double click me to play! {#yellow<}

# multiple lines
cmi setmotd {#green}This is line one \n {#orange} and this is line two
```

More info:
https://www.zrips.net/cmi/hex-colors/
